cognite/client/data_classes/data_modeling/instances.py

```diff
@@ -465,7 +464,8 @@ def to_pandas(  # type: ignore [override]
             if not extra:
                 prop_df.columns = prop_df.columns.str.removeprefix("{}.{}/{}.".format(*view_id.as_tuple()))
                 if isinstance(self, TypedInstance):
-                    prop_df.rename(columns=_get_descriptor_property_name_mapping(type(self)), inplace=True)
+                    attr_name_mapping = self._get_descriptor_property_name_mapping()
```

Comment:
If you do the remapping before `removeprefix(..)` there are no name conflicts (assuming you pad lookup with space.view/version)

Reply:
The renaming addresses conflicts with the base properties, not other properties within the prop object (that necessarily does not have conflicts).

Reply:
Oh right